src/v3/widgetrc.ts

```diff
@@ -68,7 +68,7 @@ const common: Partial<WidgetOptions> = {
     // autoFocus: true,
     // hideSignOutLinkInMFA: true,
     // mfaOnlyFlow: true,
-    rememberMe: false,
+    rememberMe: true,
```

Comment:
Just checking this is the "correct" default state

Reply:
Yessir! In [v2](https://github.com/okta/okta-signin-widget/blob/master/src/models/Settings.ts#L84) it is default to true, so I updated this to reflect that in Gen3